src/test/java/Utils/TypeBrowser.java

```diff
@@ -0,0 +1,7 @@
+package Utils;
+
+public class TypeBrowser {
+	
+	protected static final String browser = System.getProperty("exeBrowser","chrome");
+	
+}
```

Comment:
why do you need this class

Reply:
Clase removida. Se ajusta clase getDriver para soportar lla tarea de TypeBrowser